src/MassTransit.ActiveMqTransport/Contexts/ActiveMqSessionContext.cs

```diff
@@ -20,6 +20,7 @@ namespace MassTransit.ActiveMqTransport.Contexts
     using GreenPipes;
     using GreenPipes.Payloads;
     using Logging;
+    using Microsoft.Extensions.Logging;

```

Comment:
I am talking about this. MassTransit already has a simple logging abstraction and just enough adapters to cover the logging needs. Instead, this PR introduces the logging library from Microsoft and it requires a package to be installed even if my app doesn't use Microsoft logging implementation.